units/src/amount/signed.rs

```diff
@@ -380,12 +380,7 @@ impl FromStr for SignedAmount {
 
     /// Parses a string slice where the slice includes a denomination.
     ///
-    /// If the string slice is zero or negative zero, then no denomination is required.
-    ///
-    /// # Returns
-    ///
-    /// The amount if the string amount and denomination parse successfully,
-    /// otherwise returns `Err(ParseError)`.
+    /// If the returned value is zero or negative zero, then no denomination is required.
```

Comment:
In c2fee62a056c67b39a0dff336b5d22f8aa01f541:

Maybe you should say "If the string represents the zero value, then no denomination is required." Or "if the returned value *would be* zero or negative zero, then no denomination is required".

Reply:
Used the second suggestion verbatim, thanks.